server/src/mindustry/server/ServerControl.java

```diff
@@ -35,16 +36,18 @@
 import static mindustry.Vars.*;
 
 public class ServerControl implements ApplicationListener{
-    private static final int roundExtraTime = 12;
-    private static final int maxLogLength = 1024 * 512;
-
     protected static String[] tags = {"&lc&fb[D]&fr", "&lb&fb[I]&fr", "&ly&fb[W]&fr", "&lr&fb[E]", ""};
     protected static DateTimeFormatter dateTime = DateTimeFormatter.ofPattern("MM-dd-yyyy HH:mm:ss"),
         autosaveDate = DateTimeFormatter.ofPattern("MM-dd-yyyy_HH-mm-ss");
 
     public final CommandHandler handler = new CommandHandler("");
     public final Fi logFolder = Core.settings.getDataDirectory().child("logs/");
 
+    public boolean inExtraRound;
+    public Task lastTask;
```

Comment:
Implementation detail, confusing and should not be public. What is a "last task"? What does it refer to? Can it be null? What would canceling it do? When does accessing it change anything at all? None of these are clear from the variable alone.